configs/fine-tune-llm_v2/gce.yaml

```diff
@@ -0,0 +1,13 @@
+head_node_type:
+  name: head
+  instance_type: n2-standard-8
+auto_select_worker_config: true
+worker_node_types: []
+
+gcp_advanced_configurations_json:
+  instance_properties:
+    disks:
+      - boot: true
+        auto_delete: true
+        initialize_params:
+          disk_size_gb: 250
```

Comment:
I still want to know why you need this adv config here

Reply:
Example GCE job that ran out of space: https://console.anyscale-staging.com/o/anyscale-internal/jobs/prodjob_qtghhfvl3i4sehsuyl3744aykn

I can try launching a job without this to see if it is still seeing this error.

Is this a nit polish item or is it blocking for OA?

Reply:
This is a blocking item now, we are hard-rejecting all advanced configs in OA due for security -- can you see if you launch w/o this to see if you still see this error, and then follow up & fix it?